third_party/nvfuser/csrc/ir_nodes.cpp

```diff
@@ -2405,11 +2423,18 @@ std::string TensorDomain::toInlineString(int indent_size) const {
   return toString(indent_size);
 }
 
-void TensorDomain::setContiguity(const std::vector<bool>& contig) {
+void TensorDomain::setContiguity(
+    const std::vector<c10::optional<bool>>& contig) {
   TORCH_INTERNAL_ASSERT(
-      noBroadcasts(getMaybeRFactorDomain()).size() == contig.size(),
-      "Invalid contiguity vector: ",
-      contig);
+      getMaybeRFactorDomain().size() == contig.size(),
+      "Invalid contiguity vector");
```

Comment:
nit: "invalid size of contiguity vector"

Reply:
fixed